cmd/res/configuration.yaml

```diff
@@ -38,6 +38,7 @@ Device:
     Interval: "1h"
 
 Driver:
+  EnableRtspServer: "true"
```

Comment:
Is this backwards compatible or will upgrading from v3.0 to v3.1 lead to a disabled RTSP server?

Reply:
upgrading to 3.1  would not lead to a disabled rtsp server. 3.0 just will always have rtsp started  by the entrypoint.sh script, while 3.1 just adds the ability to enable or disable rtsp server.

Reply:
I don't think so. The entrypoint.sh of 3.0 will be replaced by that of 3.1 which [should not start RTSP](https://github.com/edgexfoundry/device-usb-camera/pull/268#discussion_r1296004827). When upgrading, the configuration file will not have `EnableRtspServer: "true"` so it will be the zero-value of the boolean instead which is `EnableRtspServer: "false."`

Reply:
This config setting should be a bool not a string, that way it doesn't have to be parsed and has a default value of `false`. Also for backward compatibility it should be `DisableRtspServer` since currently the RTSP Servicer is enabled by default. Please make this change.

Reply:
Oh, I didn't even notice that it was a string rather than a boolean.

Reply:
the SDK only return map[string]string for driver configurations. "

Reply:
> the SDK only return map[string]string for driver configurations. "

Right and the service doesn't have any custom config... So then the code here must consider empty string as `true` to not break backward compatibility rather then return a parse error.
https://github.com/edgexfoundry/device-usb-camera/blob/860b2ed41c63812ea8e78996a55d34ef8e22b126/internal/driver/driver.go#L119-L122

Reply:
Being addressed in https://github.com/edgexfoundry/device-usb-camera/pull/270